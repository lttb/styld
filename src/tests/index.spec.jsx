import renderer from 'react-test-renderer'

import React from 'react'
import App from './App'


it('renders correctly', () => {
  const tree = renderer.create(<App />).toJSON()

  expect(tree).toMatchSnapshot()
})
