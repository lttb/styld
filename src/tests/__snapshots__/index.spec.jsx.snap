exports[`test renders correctly 1`] = `
<div
  className="app-0-0">
  <header
    className="header-0-1">
    <h1
      className="h1-0-6">
      Just H1
    </h1>
    <h1
      className="h1_text-0-7 h1-0-6 text-0-5">
      Force test
    </h1>
  </header>
  <section
    className="section_content-0-4 section-0-3"
    data-name="content">
    <p
      className="p-0-8 text-0-5 h1-0-6">
      compose multiple classes test
    </p>
    <button
      className="button-0-9">
      primitive test
    </button>
    <button
      className="button-0-33 button-0-31">
      dynamic primitive test
    </button>
  </section>
</div>
`;
